client_src/js/serverAliveCheck.js

```diff
@@ -33,7 +33,13 @@ function checkUpdateAndNotify(updateResponse) {
     } catch (e) {
       parsedStorageReleaseInfo = null;
     }
-
+    // If the update button is available, we are on the system page. React to available updates accordingly.
+    let forceCheckForUpdatesBtn = document.getElementById(
+        "checkUpdatesForOctoFarmBtn"
```

Comment:
Repeated string literal, should be made constant. Or you can save the queried element. Your choice.